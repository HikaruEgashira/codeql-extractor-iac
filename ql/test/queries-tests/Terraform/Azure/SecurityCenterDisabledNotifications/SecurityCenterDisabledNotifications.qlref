security/Terraform/Azure/SecurityCenterDisabledNotifications.ql