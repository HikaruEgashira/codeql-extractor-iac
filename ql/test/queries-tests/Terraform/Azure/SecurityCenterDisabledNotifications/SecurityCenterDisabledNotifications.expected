| security_center.hcl:5:1:9:1 | resource azurerm_security_center_contact contact | disabled notifications |
